src/python_framework_v02/troute/nhd_io.py

```diff
@@ -680,6 +682,9 @@ def get_usgs_from_time_slices_folder(
           the interpolated values are truncated so that the first value returned
           corresponds to the first center date of the first provided file.
     """
+    max_fill_1min = data_assimilation_parameters.get("data_assimilation_interpolation_limit", 59)
+    qc_trehsh = data_assimilation_parameters.get("QC_threshold", 1)
```

Comment:
This is a spelling error we made in the original notes. Let's call this `qc_threshold` or `qc_thresh` for short, if needed.